### PR TITLE
new(content): added Falco 0.38.1 blog post.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind content

**Any specific area of the project related to this PR?**

/area blog

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/blog/falco-0-38-1/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+author: Federico Di Pierro
+slug: falco-0-38-1
+tags: ["Falco","Release"]
+---
+
+Today we announce the release of **Falco 0.38.1** 🦅!
+
+## Fixes
+
+Falco's 0.38.1 is a patch release aimed at addressing a few important bugs. It includes the following:
```
Comment (LucaGuerra) [inline content/en/blog/falco-0-38-1/index.md:13]:
```suggestion
Falco's 0.38.1 is a patch release aimed at addressing a few important bugs. It includes the following fixes:
```

**Review by leogr — APPROVED** [review]
Just 

/hold for the release

File: content/en/blog/falco-0-38-1/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+* **A Falco crash while running with plugins and metrics enabled has been solved** (https://github.com/falcosecurity/falco/issues/3229)
+* Falco `-p` output format option can be passed to plugin events while `-pc` and `-pk` can only be used for syscall sources (https://github.com/falcosecurity/falco/pull/3239)
```
Comment (LucaGuerra) [inline content/en/blog/falco-0-38-1/index.md:16]:
```suggestion
* Falco `-p` output format option can now be passed to plugin events while `-pc` and `-pk` can only be used for syscall sources. Fixes an issue that could result in Falco exiting with `LOAD_ERR_COMPILE_OUTPUT` on startup with k8s clusters that had `-pk` and audit enabled (https://github.com/falcosecurity/falco/pull/3239)
```

File: content/en/blog/falco-0-38-1/index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+* Fixed signed compare operator in libs (https://github.com/falcosecurity/falco/issues/3245)
```
Comment (LucaGuerra) [inline content/en/blog/falco-0-38-1/index.md:17]:
```suggestion
* Fixed an issue that could prevent the integer compare operators `<`, `<=`, `>`, `>=` in rules from working properly (https://github.com/falcosecurity/falco/issues/3245)
```

**Comment by FedeDP** [comment]
Thanks luca!

**Pull request merged**